scripts/ci/selective_ci_checks.sh

```diff
@@ -115,9 +115,9 @@ function output_all_basic_variables() {
     initialization::ga_output default-helm-version "${HELM_VERSION}"
 
     if [[ ${FULL_TESTS_NEEDED_LABEL} == "true" ]]; then
-        initialization::ga_output postgres-exclude '[{ "python-version": "3.6" }]'
-        initialization::ga_output mysql-exclude '[{ "python-version": "3.7" }]'
-        initialization::ga_output sqlite-exclude '[{ "python-version": "3.8" }]'
+        initialization::ga_output postgres-exclude '[{ "python-version": "3.6" }, { "python-version": "3.9" }]'
+        initialization::ga_output mysql-exclude '[{ "python-version": "3.7" }, { "python-version": "3.9" }]'
+        initialization::ga_output sqlite-exclude '[{ "python-version": "3.7" }, { "python-version": "3.8" }]'
```

Comment:
Curious, why is 3.9 being excluded for postgres/mysql and 3.7 now being excluded for sqlite?

Reply:
Somewhat arbitrary. We just want to save build/test time so we do not run full.matrix of tests. This is a way to test some 'representative' combinations.